src/index.ts

```diff
@@ -1,27 +1,21 @@
-import { getInput, info, warning, error } from '@actions/core'
+import { getInput, info, warning, error, setFailed } from '@actions/core'
 import { getOctokit, context } from '@actions/github'
+import { Context } from '@actions/github/lib/context'
 import * as github from './util/github'
 import Badge from './util/badge'
 import _ from 'lodash'
 import Resolver from './Resolver'
 
 const token = getInput('token')
+const isDebug = getInput('debug') === 'true'
 
-if (context.eventName !== github.Event.PULL_REQUEST) {
-  error(`Badger does not support '${context.eventName}' actions.`)
-} else if (context.payload.action !== github.PullRequestAction.OPENED) {
-  warning(`Skipping Badger, cannot handle '${context.action}' events.`)
-} else if (!token) {
-  error(`Authentication token not provided.`)
-} else {
-  const resolver = new Resolver(context)
-
-  const body = context.payload.pull_request.body
-  const prefix = getInput('prefix')
-  const suffix = getInput('suffix')
+if (isDebug) {
+  console.log(`********************************\nGitHub Context\n\n${JSON.stringify(context)}\n\n********************************`)
```

Comment:
Could just import `debug` from `@actions/core` and use that for automatic `isDebug` coverage 😁